prestoadmin/prestoclient.py

```diff
@@ -152,9 +155,10 @@ def get_response_from(self, uri):
         parts[0] = None
         parts[1] = None
         location = urlparse.urlunsplit(parts)
-
         conn = self._get_connection()
-        conn.request("GET", location)
+        headers = {"X-Presto-User": self.user}
+        self._add_auth_headers(headers)
```

Comment:
I would rather prefer next code

```
if self.coordinator_config.use_ldap():
     auth_headers = create_auth_headers(self.coordinator_config.get_ldap_user(), self.coordinator_config.get_ldap_password())
     headers.update(auth_headers)
```

Once you change the method signature, it can be easily unit tested. 
I would follow the same approach for unit testing as we do in Airlift (https://github.com/airlift/airlift/commit/98f6ac6d3a13efdea1dd79176f8d5fe2f6eeed3b#diff-17f1891e5fe78cabf020e0a17bfdbfc4R39): 


https://www.ietf.org/rfc/rfc2617.txt
```
create_auth_headers("Aladdin", "open sesame") == {"Authorization", "Basic QWxhZGRpbjpvcGVuIHNlc2FtZQ=="} 
```